modules/user_accounts/php/edit_user.class.inc

```diff
@@ -327,24 +338,23 @@ class Edit_User extends \NDB_Form
                 //get existing sites for examiner
                 $prev_sites = $DB->pselect(
                     "SELECT centerID
-             FROM examiners_psc_rel epr
-             WHERE examinerID=:eid",
+                            FROM examiners_psc_rel epr
+                            WHERE examinerID=:eid",
                     array("eid" => $examinerID)
                 );
-
-                //get sites where user is already an examiner at for compare
-                foreach ($prev_sites as $row => $center) {
-                    array_push($ex_prev_sites, $center['centerID']);
-                }
+            }
```

Comment:
Sorry, I think this should be on line 350 instead of here. The original file had the next foreach loop inside of the else statement.